### PR TITLE
Revamped account binding flow and messages

### PR DESCRIPTION
**Pull request by andela-Kshittu**
* Delete original prompt, post-binding to reflect the new state

* Redesigned link unfurl account binding prompt
screen shot 2018-08-16 at 10 39 33 am

* Redesigned slash command account binding prompt 
screen shot 2018-08-16 at 10 39 56 am

NOTE: We've added a new scope `chat:write:bot`. since merging will auto deploy to master, we should update scopes and permission for our slack apps and reinstall the apps before merging this.

### CODE REVIEW

File: server/models/authmessage.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Data.World Slack Application
```
Comment (rflprr) [inline server/models/authmessage.js]: Because you just changed to lowercase everywhere...

File: server/controllers/unfurl.js [hunk 2]
```diff
@@ -418,31 +413,29 @@ const handleJoinedChannelEvent = async event => {
...
-    // TODO : Should delete all DW active subscriptions by this user
-
-    // delete from DB
-    await Subscription.destroy({
-      where: { slackUserId: user.slackId }
-    });
-    // delete the user
-    await User.destroy({
-      where: { slackId: user.slackId }
+  try {
```
Comment (rflprr) [inline server/controllers/unfurl.js]: I'm curious about this. I think it's great that we clean things up on our end. However, I'd like to make sure that if 1) we don't get this event or 2) something goes wrong and clean-up is partially complete, that there are no side-effects.

Specifically, I want to make sure that, if the app is reinstalled, it won't be in a corrupted state.

**Comment by rflprr** [comment]
@andela-Kshittu New scope added and app reinstalled.

File: server/models/authmessage.js (hunk 1 above)
Comment (andela-Kshittu) [reply server/models/authmessage.js]: Ahhh..Good catch! thanks.

File: server/controllers/unfurl.js (hunk 2 above)
Comment (andela-Kshittu) [reply server/controllers/unfurl.js]: We are performing 3 actions at that point. (edited)
1. delete all subscriptions for each user in the team
2. delete users  in the team
3. delete the team

If *1* fails or completes partially, When the app is re-installed the subscriptions not deleted will only be active and show up in slack if corresponding DW subscriptions are still active.

If *2* fails or completes partially,  When the app is re-installed the user records not deleted will be active if access has not been revoked from DW, if access had been revoked, account binding will be requested and user record will be updated

If *3* fails or completes partially,  When the app is re-installed the team record not deleted will get updated.

**Pull request merged**